packages/framework/request-handler/src/requestHandlers.ts

```diff
@@ -38,7 +38,11 @@ export const rootDataStoreRequestHandler = async (request: IRequest, runtime: IC
     try {
         // getRootDataStore currently throws if the data store is not found
         const rootDataStore = await runtime.getRootDataStore(id, wait);
-        return rootDataStore.IFluidRouter.request(requestParser.createSubRequest(1));
+        try {
```

Comment:
Although this works, I think it's more easily readable as two sequential try/catches rather than nesting.  I.e. try getRootDataStore, returned undefined if error, THEN separately try issuing the subrequest and return 500 if error.